webhook-service/src/test/scala/io/renku/webhookservice/hookfetcher/ProjectHookFetcherSpec.scala

```diff
@@ -51,118 +59,85 @@ class ProjectHookFetcherSpec
     "return the list of hooks of the project - personal access token case" in new TestCase {
```

Comment:
Similarly as above, we could merge this and the next tests and use the `accessTokens` generator.